lib/elixir/pages/Naming Conventions.md

```diff
@@ -0,0 +1,90 @@
+# Naming Conventions
+
+## Naming modules
+
+Elixir module names are written in "upper camel case", like `OptionParser`.
+
+Capital letters are kept in abbreviations, like `ExUnit.CaptureIO` or `Mix.SCM`.
```

Comment:
I believe this was a bug in very early Elixir versions. MyApp.CLI should work just fine.